lib/services/remote_config.dart

```diff
@@ -33,15 +33,15 @@ class RemoteConfigService {
     try {
       await _remoteConfig.setDefaults(defaults);
       await _fetchAndActivate();
-    } on FetchThrottledException catch (e) {
+    } on Exception catch (e) {
```

Comment:
Revert back to FetchThrottledException, as every other exception will be caught by the cath block below.